src/lib/dependencies/nuget.js

```diff
@@ -66,7 +66,38 @@ function getDependenciesFromGithubRepo (githubRepo, githubAccessToken) {
     });
 }
 
+function dependenciesStats (file) {
+  if (file.name === 'packages.config') {
+    const xml = new xmldoc.XmlDocument(file.text);
+    return packagesConfigDependenciesStats(xml);
+  }
+  if (file.name.indexOf('.csproj') !== -1) {
+    const xml = new xmldoc.XmlDocument(file.text);
+    return csprojDependenciesStats(xml);
+  }
+  return [];
+}
+
+function isDependencyFile (file) {
+  if (file.name === 'packages.config' || file.name.indexOf('.csproj') !== 1) {
+    return true;
+  }
+}
+
+function detectProjectName (file) {
+  if (file.name.indexOf('.csproj') !== -1) {
+    const xml = new xmldoc.XmlDocument(file.text);
+    const searchAssemblyTitles = xml.childrenNamed('PropertyGroup').map(itemGroup => itemGroup.childrenNamed('AssemblyTitle'));
```

Comment:
What do you think about using `<PackageId>` here instead of `<AssemblyTitle>`? The `AssemblyTitle` is used to define the DLL name, and the `PackageId` is used to define the name in NuGet repository?

Also if either of those properties are missing in the file contents, then the `FileName` in `{FileName}.csproj` is automatically used.

Reply:
If it's a private project not published on NuGet, what do you think is the key likely to be used?

Reply:
I think falling back to the filename without .csproj is probably a good first step. We can always get feedback and adjust it!